src/scss/_api-symbol.scss

```diff
@@ -1,3 +1,4 @@
+/* purgecss start ignore */
 $size-small: 9px;
```

Comment:
A comment as to why we want to PurgeCSS to ignore this file would be good so the ignore-comment doesn't accidentally get removed by someone else.